playground/src/import-url/import-url.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+require('./import-url.scss');
+
+import events from '../events/events.js';
+const fetch = require('node-fetch');
```

Comment:
Careful: this is a client-side/browser bundle so the Fetch API will be available and doesn't need to be bundled.